<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="ErrorValidationTest(failed)">
    <groups>
      <run>
        <include name="ErrorHandling"/>
      </run>
    </groups>
    <classes>
      <class name="Rahulshettyacademy.rahulshettyacademy.tests.ErrorvalidationTest">
        <methods>
          <include name="teardown"/>
          <include name="launchapp"/>
          <include name="LoginErrorValidation"/>
        </methods>
      </class> <!-- Rahulshettyacademy.rahulshettyacademy.tests.ErrorvalidationTest -->
    </classes>
  </test> <!-- ErrorValidationTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
